packages/AnimationsEditor-Core.package/AnimationsEditorProperty.class/instance/playAnimationWithStartEnd..st

```diff
@@ -0,0 +1,16 @@
+animation
+playAnimationWithStartEnd: aStartEndProperty
+	| animations comp |
+	
+	(self isKeyframeExistent)
+		ifTrue: [
```

Comment:
refactoring to a seperate method?

Reply:
what part would you like to seperate? we don't see it